frontend/src/components/GovtMasthead/GovtMasthead.tsx

```diff
@@ -21,36 +21,71 @@ import { GovtMastheadItem } from './GovtMastheadItem'
 export const GovtMasthead = (): JSX.Element => {
   const { isOpen, onToggle } = useDisclosure()
 
+  // Custom function for collapsing/expanding the header,
+  // because in mobile you tap the whole header, on desktop only on the link
+  const isMobile =
+    useBreakpointValue({
+      base: true,
+      xs: true,
+      md: false,
+    }) ?? true
+
+  interface HeaderBarProps {
+    isMobile: boolean
+    children: React.ReactNode
+  }
+
+  interface HowToIdentifyProps {
+    isMobile: boolean
+  }
+
+  const HeaderBar = ({ isMobile, children }: HeaderBarProps): JSX.Element => {
+    const styleProps = {
+      bg: 'neutral.200',
+      py: { base: '0.5rem', md: '0.375rem' },
+      px: { base: '1.5rem', md: '1.75rem', lg: '2rem' },
+      textStyle: { base: 'legal', md: 'caption-2' },
+      display: 'flex',
+      width: '100%',
+    }
+    return (
+      <Flex {...styleProps} {...(isMobile ? { onClick: onToggle } : {})}>
+        {children}
+      </Flex>
+    )
+  }
+
+  const HowToIdentify = ({ isMobile }: HowToIdentifyProps): JSX.Element => {
+    const howToIdentifyProps = {
+      tabIndex: 0,
+      ariaLabel: 'Click to expand masthead for more information',
+      onClick: onToggle,
+    }
+
+    return (
+      <Link {...(isMobile ? {} : howToIdentifyProps)}>
```

Comment:
Might want to do the same for this, where it is a real `Link` component in mobile, and `Text` in non-mobile

Reply:
will do - the opposite though. :P `Text` in mobile and `Link` otherwise.